azure-pipelines.yml

```diff
@@ -29,13 +29,10 @@ stages:
         zivid_versions: ["${{ variables['newest_zivid_version'] }}"]
         ros_distros: ['ros:kinetic-ros-base-xenial']
         compilers: ['g++-7']
-# Currently only newest_zivid_version is supported. Azure Pipelines requires at least one job under
-# jobs, so parameters.zivid_versions cannot be empty. Re-enable this when we support >0 old versions.
-#
-#  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
-#    jobs:
-#    - template: continuous-integration/azure-pipelines/templates/build_and_test.yml
-#      parameters:
-#        zivid_versions: ['']
-#        ros_distros: ['ros:melodic-ros-base-bionic', 'ros:kinetic-ros-base-xenial']
-#        compilers: ['g++-7']
+  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
+    jobs:
+    - template: continuous-integration/azure-pipelines/templates/build_and_test.yml
+      parameters:
+        zivid_versions: ['2.0.0+7c4918cf-14']
+        ros_distros: ['ros:melodic-ros-base-bionic', 'ros:kinetic-ros-base-xenial']
+        compilers: ['g++-7']
```

Comment:
What's the reason for testing only for one compiler on the older version(s)?
To save build time on the older version, also when there are more of them?

Reply:
To avoid combinatoric explosion I chose only oldest supported compiler for old SDK. Does not matter that much now since there is only 1 old version, but I think it is sufficient to test just the newest SDK for all the compilers.

Reply:
Ok, makes sense :+1: